common/djangoapps/student/views.py

```diff
@@ -573,6 +575,13 @@ def dashboard(request):
         and has_access(request.user, 'view_courseware_with_prerequisites', enrollment.course_overview)
     )
 
+    # get the programs associated with courses being displayed.
+    # pass this along in template context in order to render additional
+    # program-related information on the dashboard view.
+    course_programs = {}
+    if ProgramsApiConfig.current().is_student_dashboard_enabled:
```

Comment:
i think it would be better to define a function in the programs djangoapp for  `is_student_dashboard_enabled` and have that function internally use the config model.   it's better if we do not import models across djangoapps.